server/handlers/meshery_pattern_handler.go

```diff
@@ -381,6 +381,43 @@ func (h *Handler) CloneMesheryPatternHandler(
 	fmt.Fprint(rw, string(resp))
 }
 
+// swagger:route POST /api/pattern/catalog/publish PatternsAPI idPublishCatalogPatternHandler
+//
```

Comment:
```suggestion
// Handle Publish for a Meshery Pattern
//
```

nitpick. 
Would fix this: 

![Screenshot 2022-10-12 at 9 33 35 PM](https://user-images.githubusercontent.com/73700530/195876176-e5064f1e-e895-4d63-bc8a-e3a60a9d0f83.png)

Reply:
Whoops these comments slipped my mind. Thanks for reviewing @theBeginner86. Will fix these